src/Timers.h

```diff
@@ -105,6 +189,14 @@ class Timers
   // functions
   std::vector<Timer> LoadTimers();
   static bool FindTagInTimerTags(std::string tag, std::string tags);
+  static std::string ConvertToAutoTimerTag(std::string tag);
+  std::vector<AutoTimer> LoadAutoTimers();
+  bool CanAutoTimers() const;
+  bool IsAutoTimer(const PVR_TIMER &timer);
```

Comment:
You should always check whether methods can be made const (not checked for this particular case, though).